src/common.js

```diff
@@ -367,11 +369,13 @@ function wrap(cmd, fn, options) {
           if (e.msg === 'earlyExit') {
             retValue = e.retValue;
           } else {
+            /* istanbul ignore next */
```

Comment:
I'll try it tonight and see if I see a difference in either line or branch coverage